### DISCUSSION TITLE
Tab completion?

### DISCUSSION THREAD

**Discussion by swrobel** [ruby/debug#1049] [Q&A]
Is auto-suggestion/tab completion for methods supposed to work as it does when using `binding.irb`?

Using `binding.irb`
<img width="175" alt="Screenshot 2023-12-14 at 3 16 59 PM" src="https://github.com/ruby/debug/assets/157270/47fec0c8-e005-43df-873b-6a98742c2a8e">

Using `binding.b`
<img width="255" alt="Screenshot 2023-12-14 at 3 19 13 PM" src="https://github.com/ruby/debug/assets/157270/e16f602e-a423-41a6-a38c-623bcede2a5e">

**Comment by ko1** [comment]
Not yet (planned)
